src/Microsoft.Identity.Client/ITokenCache.cs

```diff
@@ -89,109 +89,109 @@ public interface ITokenCache
+        ///// <see cref="SerializeMsalV3"/>/<see cref="DeserializeMsalV3"/> is compatible with other MSAL libraries such as MSAL for Python and MSAL for Java.
+        ///// </remarks>
+        //byte[] SerializeMsalV2();
+
+        ///// <summary>
+        ///// Deserializes the token cache to the MSAL.NET 2.x cache format, which is compatible with ADAL.NET v4 and other MSAL.NET v2 applications.
+        ///// If you need to maintain SSO between an application using ADAL 3.x or MSAL 2.x and this application using MSAL 3.x,
+        ///// you might also want to serialize and deserialize with <see cref="SerializeAdalV3"/>/<see cref="DeserializeAdalV3"/> or <see cref="SerializeMsalV2"/>/<see cref="DeserializeMsalV2"/>,
+        ///// otherwise just use <see cref="SerializeMsalV3"/>/<see cref="DeserializeMsalV3"/>.
+        ///// </summary>
+        ///// <param name="msalV2State">Byte stream representation of the cache</param>
+        ///// <remarks>
+        ///// <see cref="SerializeMsalV3"/>/<see cref="DeserializeMsalV3"/> is compatible with other MSAL libraries such as MSAL for Python and MSAL for Java.
+        ///// </remarks>
+        //void DeserializeMsalV2(byte[] msalV2State);
+
+        ///// <summary>
+        ///// Serializes the token cache to the ADAL.NET 3.x cache format.
+        ///// If you need to maintain SSO between an application using ADAL 3.x or MSAL 2.x and this application using MSAL 3.x,
+        ///// you might also want to serialize and deserialize with <see cref="SerializeAdalV3"/>/<see cref="DeserializeAdalV3"/> or <see cref="SerializeMsalV2"/>/<see cref="DeserializeMsalV2"/>,
+        ///// otherwise just use <see cref="SerializeMsalV3"/>/<see cref="DeserializeMsalV3"/>.
+        ///// </summary>
+        ///// <returns>Byte stream representation of the cache</returns>
+        ///// <remarks>
+        ///// <see cref="SerializeMsalV3"/>/<see cref="DeserializeMsalV3"/> is compatible with other MSAL libraries such as MSAL for Python and MSAL for Java.
+        ///// </remarks>
+        //byte[] SerializeAdalV3();
+
+        ///// <summary>
+        ///// Deserializes the token cache to the ADAL.NET 3.x cache format.
+        ///// If you need to maintain SSO between an application using ADAL 3.x or MSAL 2.x and this application using MSAL 3.x,
+        ///// you might also want to serialize and deserialize with <see cref="SerializeAdalV3"/>/<see cref="DeserializeAdalV3"/> or <see cref="SerializeMsalV2"/>/<see cref="DeserializeMsalV2"/>,
+        ///// otherwise just use <see cref="SerializeMsalV3"/>/<see cref="DeserializeMsalV3"/>.
+        ///// </summary>
+        ///// <param name="adalV3State">Byte stream representation of the cache</param>
+        ///// <remarks>
+        ///// <see cref="SerializeMsalV3"/>/<see cref="DeserializeMsalV3"/> is compatible with other MSAL libraries such as MSAL for Python and MSAL for Java.
+        ///// </remarks>
+        //void DeserializeAdalV3(byte[] adalV3State);
```

Comment:
[obsolete] comments should probably be something like what you have in the remarks for SetBeforeAccessAsync and SetAfterAccessAsync ?
(applies to all commented out methods)

Reply:
Agreed.  I submitted this to get a test build other than on my machine.  VS 16.1 borked my ability to fully build locally.  I think the integration tests are broken due to Bogdan saying he added some traces that were hanging the tests.  But I will investigate further as well.